benchmark/buffers/buffer-equals.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  size: [0, 16, 512, 4096, 16386],
```

Comment:
It should be sufficient to just keep e.g., `0`, `512` and `16386`. The other options are just intermediate steps.

Reply:
FWIW I copied this from `buffer-compare.js`, does that mean that should be changed as well?

Reply:
That would IMO be ideal :+1: